config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -344,7 +344,7 @@ presubmits:
         - --repo=k8s.io/kubernetes=$(PULL_REFS)
         - --repo=k8s.io/release
         - --upload=gs://kubernetes-jenkins/pr-logs
-        - --timeout=200
+        - --timeout=320
```

Comment:
adding two hours here seems a little excessive, to the point where I would not really expect this test to complete in time to inform some merges ... was the test time really expected to jump this much?

Reply:
This job has taken 2h 30m before my PR was merged, and about 30 tests are added.
- about 6 tests take 4min for each to make sure that something not happens
- the rest of tests take 30~1.5min for each

I found that no test takes unusual amount of time.
It may takes 3h 0~30min roughly but this is timeout, so I just wanted to add some extra time just in case something is wrong or other tests are added.

I think we can increase the timeout for now and find a way to decrease it, as this job rarely succeeds now.

(I think this job does not run tests in parallel even if it does not run serial tests. Maybe we can fix it first?)

Reply:
https://github.com/kubernetes/test-infra/pull/30137

maybe running the job in parallel may resolve the issue. Could you PTAL?

Reply:
Or we could remove non-alpha tests from this job. Will propose another PR for it.

Reply:
`pull-kubernetes-e2e-cos-alpha-features` has been failing for about a week. (since Jul 19?)

I would suggest bumping the timeout or excluding SidecarContainers tests from this job until we find a proper way to test this amount of specs faster.

@liggitt

Reply:
> I would suggest bumping the timeout or excluding SidecarContainers tests from this job until we find a proper way to test this amount of specs faster.

I'll let testing folks (@aojea @pohly @BenTheElder) and node folks (@SergeyKanzhelev) weigh in on which is preferred

Reply:
presubmit should not take long, we have periodic jobs for long running jobs.

I didn't take a look to the job itself but as soon as feature graduate we should be removing tests from this job, is there is a problem that we are not graduating features or that we forget to update the tests?
if we are not graduating features in alpha and keep adding more features in alpha that is worrysome